### ISSUE TITLE
Confusing Speech output on Shortcuts to Open/Close Panels

### ISSUE THREAD

**Issue by jmwmusic** [musescore/MuseScore#30462]
### Issue type

Accessibility issue (e.g. for keyboard-only or screen reader users)

### Description with steps to reproduce

When you press any of the Shortcuts such as F7 for Layout, F8 for Properties, F9 for Palettes, etc, the first Press announces the name e.g. Palettes, but it does not say whether the palette is Opening or Closing.  Confusingly if you Press the key a second time there is no speech output at all.  It does not seem to matter whether the Panel was Open or Closed before the first press of the key.

Ideally speech output should always give feedback as to the action that has occurred rather than confirming a key pressed.
so, in this instance for palettes, it should say "Palette Open" and "Palette Closed".

### Supporting files, videos and screenshots

N/A

### In which versions of MuseScore Studio is this issue present?

4.6.1

### Regression

I was unable to check

### Operating system

Windows 11 with NVDA

### Checklist

- [x] This report follows the [guidelines](https://github.com/musescore/MuseScore/wiki/Reporting-bugs-and-issues) for reporting bugs and issues
- [x] I have verified that this issue has not been logged before, by searching the [issue tracker](https://github.com/musescore/MuseScore/issues) for similar issues
- [x] I have attached all requested files and information to this report
- [x] I have attempted to identify the root problem as concisely as possible, and have used minimal reproducible examples where possible

**Comment by jmwmusic** [comment]
This is continuing to create confusion for new MuseScore users working with a screen reader.  They are not aware they are closing the palette when they press F9, instead expecting it to jump to it. Even those reading the handbook report confusion as it isn't clear it is a toggle and speech does not confirm Open or Closed.  See also: #20099 re F9 no longer taking focus to palettes and #12873 Ctrl F9 not taking focus to palette search.
